tests/chainer_tests/links_tests/normalization_tests/test_batch_normalization.py

```diff
@@ -53,6 +55,19 @@ def test_forward_gpu(self):
         self.link.to_gpu()
         self.check_forward(cuda.to_gpu(self.x))
 
+    @condition.retry(3)
+    def test_forward_cpu_volatile(self):
+        self.volatile = 'on'
+        self.link.test = True
```

Comment:
Does `BatchNormalization` or `Link` has `test` attribute? Otherwise, this assignment does not have any effect.